js/bibox.js

```diff
@@ -452,6 +452,8 @@ module.exports = class bibox extends Exchange {
     }
 
     parseOrder (order, market = undefined) {
+        if (!('order_type' in order) || !('order_side' in order) || !('createdAt' in order))
```

Comment:
actually, it would work simply to check whether `order == {}`. @kroitor do you know the proper way to do this in javascript that will survive traspiling?

Reply:
```JavaScript
if (Object.keys (order).length < 1) {
    // empty object
}
```

Reply:
done...

in fact, some of the `safeString` checks might actually be unnecessary now. though I didn't realize it earlier, these keys were missing simply because the entire order was empty :(

I guess it's better to have them than not.